README.md

```diff
@@ -0,0 +1,211 @@
+    ```
+
+*  **CentOS based image**
+    ```
+    $ git clone https://github.com/openshift/sti-ruby.git
+    $ cd sti-ruby
+    $ make build VERSION=2.0
+    ```
+
+**Notice: By omitting the `VERSION` parameter, the build/test action will be performed
+on all the supported versions of Ruby. Since we are now supporting only version `2.0`,
+you can omit this parameter.**
+
+
+Usage
+---------------------
+To build a simple [ruby-sample-app](https://github.com/jhadvig/sti-ruby-test/tree/master/2.0/test/test-app) application,
+using standalone [STI](https://github.com/openshift/source-to-image) and then run the
+resulting image with [Docker](http://docker.io) execute:
+
+*  **For RHEL based image**
+    ```
+    $ sti build https://github.com/openshift/sti-ruby.git --contextDir=/2.0/test/test-app/ openshift/ruby-20-rhel7 ruby-sample-app
+    $ docker run -p 9292:9292 ruby-sample-app
+    ```
+
+*  **For CentOS based image**
+    ```
+    $ sti build https://github.com/openshift/sti-ruby.git --contextDir=/2.0/test/test-app/ openshift/ruby-20-centos7 ruby-sample-app
+    $ docker run -p 9292:9292 ruby-sample-app
+    ```
+
+**Accessing the application:**
+```
+$ curl 127.0.0.1:9292
+```
+
+
+Test
+---------------------
+This repository also provides [STI](https://github.com/openshift/source-to-image) test framework,
+which launches tests to check functionality of a simple ruby application built on top of sti-ruby image.
+
+User can choose between testing ruby test application based on RHEL or CentOS image.
+
+*  **RHEL based image**
+
+    This image is not available as automated build in [Docker Index](https://index.docker.io).
+
+    To test a rhel7-based ruby-2.0 image, you need to run the test on a properly
+    subscribed RHEL machine.
+
+    ```
+    $ cd sti-ruby
+    $ make test TARGET=rhel7 VERSION=2.0
+    ```
+
+*  **CentOS based image**
+
+    ```
+    $ cd sti-ruby
+    $ make test VERSION=2.0
+    ```
+
+**Notice: By omitting the `VERSION` parameter, the build/test action will be performed
+on all the supported versions of Ruby. Since we are now supporting only version `2.0`
+you can omit this parameter.**
+
+
+Repository organization
+------------------------
+* **`<ruby-version>`**
+
+    * **Dockerfile**
+
+        CentOS based Dockerfile.
+
+    * **Dockerfile.rhel7**
+
+        RHEL based Dockerfile. In order to perform build or test actions on this
+        Dockerfile you need to run the action on properly subscribed RHEL machine.
+
+    * **`.sti/bin/`**
+
+        This folder contains scripts that are run by [STI](https://github.com/openshift/source-to-image):
+
+        *   **assemble**
+
+            Is used to install the sources into location from where the application
+            will be run and prepare the application for deployment (eg. installing
+            modules using npm, etc..)
+
+            Following [markers](#markers) take effect in `assemble` script:
+            - disable_asset_compilation
```

Comment:
Looks like you moved to an ENV switch but you still need to remove this marker doc.